src/SFA.DAS.FAA.Data/SavedSearch/SavedSearchRepository.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using SFA.DAS.FAA.Domain.Entities;
+using SFA.DAS.FAA.Domain.Models;
+
+namespace SFA.DAS.FAA.Data.SavedSearch
+{
+    public interface ISavedSearchRepository
+    {
+        Task<PaginatedList<SavedSearchEntity>> GetAll(DateTime dateFilter, int pageNumber, int pageSize, CancellationToken token);
+    }
+
+    public class SavedSearchRepository(IFindApprenticeshipsDataContext dataContext) : ISavedSearchRepository
+    {
+        public async Task<PaginatedList<SavedSearchEntity>> GetAll(DateTime dateFilter, int pageNumber, int pageSize, CancellationToken token = default)
```

Comment:
With pagination I assume the job is always getting the first page?

Reply:
yes, default values have been set in the controller so that it was returns page 1